server/crawler/crawler.go

```diff
@@ -259,15 +262,8 @@ func ReadPackage(
 	}
 	if !packageFound {
 		logrus.Warn("No Kurtosis package found.") // don't want to log provided package repository locator bc it's a security risk (e.g. malicious data)
-		return nil, stacktrace.NewError("No Kurtosis package found. Ensure that a package exists at '%v' with valid '%v' and '%v' files.", packageRepositoryLocator, kurtosisYamlFileName, starlarkMainDotStarFileName)
-	}
-
-	// fill it with the repository starts
-	repository, _, err := githubClient.Repositories.Get(ctx, apiRepositoryMetadata.GetOwner(), apiRepositoryMetadata.GetName())
```

Comment:
removing the call here because Im centralizing all calls in the `extractKurtosisPackageContent` method because it's used in both endpoints